test/configure-errors.js

```diff
@@ -0,0 +1,107 @@
+(function() {
+  var loopback = require('loopback');
+  var app = loopback();
+  var m = require('./init');
+  var mock = require('mock-require');
+  var PassportConfigurator = require('../lib/passport-configurator.js');
+  var passportConfigurator = new PassportConfigurator(app);
+  var passport = require('passport');
+  var assert = require('assert');
+  var request = require('supertest');
+  var sinon = require('sinon');
+  var passportAuthenticate;
+
+  //mock a strategy to work with
+  function PassportFacebookStrategy(options, verify) {};
+  PassportFacebookStrategy.prototype.authenticate = function(req, options) {};
+  mock('passport-facebook', PassportFacebookStrategy);
```

Comment:
Do all this inside a before block as described above. Add more as you need to describe what you're doing.